blocks/header/header.js

```diff
@@ -285,7 +285,9 @@ export default async function decorate(block) {
   });
 
   // hamburger for mobile
-  const hamburger = createTag('a', { class: 'nav-hamburger', role: 'button', tabindex: '0' });
+  const hamburger = createTag('a', {
+    class: 'nav-hamburger', role: 'button', tabindex: '0', 'aria-label': 'nav-mobile',
```

Comment:
Why did you change the order of these, is this significant? (real question, I'm still learning)